docs/rules/no-shadow-native-events.md

```diff
@@ -0,0 +1,51 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-shadow-native-events
+description: disallow `emits` which would shadow native html events
+---
+# vue/no-shadow-native-events
+
+> disallow `emits` which would shadow native HTML events
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> _**This rule has not been released yet.**_ </badge>
+
+## :book: Rule Details
+
+This rule reports emits that shadow native HTML events. (The `emits` option is a new in Vue.js 3.0.0+)
```

Comment:
This is not relevant for this rule, since `$emit()` calls are also reported.

```suggestion
This rule reports emits that shadow native HTML events.
```